roles/tendrl-server/tasks/firewalld.yml

```diff
@@ -0,0 +1,44 @@
+---
+
+- name: Check if firewalld is up and running
+  include_role:
+    name: tendrl-server
+    tasks_from: firewalld-check
+
+- name: Enable port for etcd
+  firewalld:
+    port=2379/tcp
+    permanent=yes
+    state=enabled
+
+- name: Enable port for graphite
+  firewalld:
+    port=2003/tcp
+    permanent=yes
+    state=enabled
+
+- name: Enable port for Graphite web
+  firewalld:
+    port=10080/tcp
+    permanent=yes
+    state=enabled
+
+- name: Enable port for http
+  firewalld:
+    port=80/tcp
+    permanent=yes
+    state=enabled
+
+- name: Enable port for https
+  firewalld:
+    port=443/tcp
+    permanent=yes
+    state=enabled
+
+- name: Enable port for grafana server
+  firewalld:
+    port=3000/tcp
+    permanent=yes
+    state=enabled
+  notify:
+    - restart firewalld
```

Comment:
Strictly speaking, we should have this notification in every task which enables a port.